backend/src/app.ts

```diff
@@ -47,6 +47,25 @@ app.get("/", (req, res) => {
   res.status(200).json("API Is Healthy");
 });
 
+// SMB2 Healt check connection to LAnd drive
```

Comment:
Is this health check route supposed to be commented out?

Reply:
Need to refactor, I found this morning that it crash the apps